### PR TITLE
Allow multiple announcement bars in Header group

### LINKED ISSUE #2583: [Navigation UX] Allow Announcement bar to be added multiple times (stacked)
The announcement bar is currently a static section - it cannot be added or removed from its current position in the header. 

User story: As a merchant I should be able to add multiple announcement bars in my header 

**Task**

- [ ] Allow the announcement bar to be added as a section within the header group
- [ ] Limit the announcement bar to the header group only 
- [ ] Confirm upgrade path for merchants.

### PR DESCRIPTION
**Pull request by metamoni**
### PR Summary: 
Allows merchants to add multiple `Announcement bar` sections to the Header group.

### Why are these changes introduced?

Fixes #2583 

### What approach did you take?
- [x] added presents to `announcement-bar` section
- [x] tested theme update in `spin`

### Visual impact on existing themes
When merchants click "Add section" in the Header group, they should be able to see an `Announcement bar` section. They should be able to add as many of these sections as they want, but only to the Header group.

_Before_
without-announcement-bar

_After_
with-announcement-bar

### Testing steps/scenarios
- [ ] Go to the [theme editor](https://os2-demo.myshopify.com/admin/themes/139923357718/editor)
- [ ] In the `Header` group, click `Add section` and verify that you see the `Announcement bar` section
- [ ] Verify that you don't see the `Announcement bar` as an option in the `Template` and `Footer` groups
- [ ] Verify that you can add multiple announcement bars to the Header
- [ ] Change the settings of each announcement bar
- [ ] Test on mobile and different browsers

### Demo links
- [Store](https://os2-demo.myshopify.com/?preview_theme_id=139923357718)
- [Editor](https://os2-demo.myshopify.com/admin/themes/139923357718/editor)

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by eugenekasimov — APPROVED** [review]
🚀

**Comment by melissaperreault** [comment]
Unsure if you are able to replicate but this new ability made me uncover some lag and jumps on the page as I browse the navigation on smaller screen. [Video reference](https://screenshot.click/11-58-z98cd-rpls2.mp4) (to watch at 0:25)

File: sections/announcement-bar.liquid [hunk 1]
```diff
@@ -146,12 +149,15 @@
...
+  "presets": [
+    {
+      "name": "t:sections.announcement-bar.presets.name",
+      "blocks": [
+        {
+          "type": "announcement"
+        }
+      ]
+    }
+  ]
```
[the remark this answers is not in the document]
Comment (metamoni) [reply sections/announcement-bar.liquid:161]: @melissaperreault I'm struggling to replicate the behaviour in your video, where the font initially appears bigger, but I do see a slight jump. I have noticed it on `main` as well, though, so I'm not sure that this issue is introduced by the stacked announcement bars. Here's a [video](https://videobin.shopify.io/v/DRo0DR) of what I'm seeing on `main`, in slow motion.

[the remark this answers is not in the document]
Comment (melissaperreault) [reply sections/announcement-bar.liquid:161]: Ok thanks! We can open an issue to investigate further!

**Pull request merged**